package.json

```diff
@@ -31,14 +31,14 @@
         "@sa11y/jest": "^0.1.4-beta.0",
```

Comment:
Looks like npm update didn't pick all latest versions. Here's what I have for Automation Components:
```
"@prettier/plugin-xml": "^0.13.0",
"@sa11y/jest": "^0.2.1",
"@salesforce/eslint-config-lwc": "^0.9.0",
"@salesforce/eslint-plugin-aura": "^1.4.0",
"@salesforce/sfdx-lwc-jest": "^0.11.0",
"eslint": "^7.19.0",
"eslint-config-prettier": "^7.2.0",
"husky": "^4.3.8",
"lint-staged": "^10.5.3",
"prettier": "^2.2.1",
"prettier-plugin-apex": "^1.8.0",
"semver": "^7.3.4"
```

Reply:
yeah. plus that jest one is causing CI to fail.